templates/index.html

```diff
@@ -56,55 +50,91 @@
 
                 <div class="row">
                     <div class="col-sm-12 form-group">
-                        <label for="Output" class='tr' key='res' size="+3"> RESULT:</label>
+                        <label for="Output" class='tr' id='res' size="+3"> RESULT:</label>
                         <textarea class="form-control tr" key='reso' type="textarea" size="+3" name="Output" id="take"
                             placeholder="" maxlength="6000" rows="5"></textarea>
                     </div>
                 </div>
 
                 <input type="text" id='takes'>
 
-                <div id=":rg" class="Am Al editable LW-avf tS-tW" hidefocus="true" aria-label="Message Body"
-                    g_editable="true" role="textbox" aria-multiline="true" contenteditable="true" tabindex="1"
-                    style="direction: ltr; min-height: 269px;" itacorner="6,7:1,1,0,0" spellcheck="false">
-                </div>
+
+
+                <form method="POST">
```

Comment:
how about adding style to this drop-down, atleast later.

Reply:
iam working on it.

Reply:
I will merge it for now, be sure to add style to this dropdown menu